ces-poc-demo/Implementation/IWeather.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ces_poc_demo.Implementation
+{
+    public interface IWeather
+    {
+
+    }
```

Comment:
If its not complete, please don't add it as part of PR

Reply:
Removed this file